libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SFSDKSalesforceSDKUpgradeManager.m

```diff
@@ -173,4 +183,12 @@ + (void)updateEncryptionForUserAccountPath:(NSString *)userAccountPath {
     [encryptedArchiveData writeToFile:userAccountPath atomically:YES];
 }
 
++ (void)upgradePasscode {
```

Comment:
Add `TODO:` to remove this in `10.0`.

Reply:
Shouldn't we remove this in `11.0`?  If we remove in `10.0` customers will not be safe to upgrade from `9.1.1` to `10.0`.

Reply:
Sure, `11.0` is fine too. Let's add a `TODO:` here so we don't forget.